server/src/controllers/temp-outfitter.es6

```diff
@@ -16,16 +15,13 @@ const email = require('../email/email-util.es6');
 const Revision = require('../models/revision.es6');
 const TempOutfitterApplication = require('../models/tempoutfitter-application.es6');
 const util = require('../services/util.es6');
+const commonControllers = require('./common.es6');
 const vcapConstants = require('../vcap-constants.es6');
 
 const tempOutfitter = {};
 
-/** Initialize our S3 bucket connection for file attachments */
-const s3 = new AWS.S3({
-  accessKeyId: vcapConstants.accessKeyId,
-  secretAccessKey: vcapConstants.secretAccessKey,
-  region: vcapConstants.region
-});
+
+const s3 = util.getS3();
```

Comment:
is this now commonController? or is there still an utility controller?

Reply:
@lauraGgit No. util is there as service, there's still common controller.